### PR TITLE
三浦拓実 EX7 解答 python

### PR DESCRIPTION
**Pull request by miuratakumiF4**
# What

- 単一数字発話の認識
- 特徴量：log-melスペクトル (64x32)
- 識別機：CNN
- モデル
`Net(
  (conv1): Conv2d(1, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
  (batchnorm1): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  (pool1): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)
  (resblock1): Sequential(
    (0): ResBlock(
      (conv): Conv2d(64, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
      (batchnorm): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    )
    (1): ResBlock(
      (conv): Conv2d(64, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
      (batchnorm): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    )
  )
  (conv2): Conv2d(64, 16, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
  (batchnorm2): BatchNorm2d(16, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  (pool2): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)
  (liner_in): Linear(in_features=2048, out_features=256, bias=True)
  (liner_out): Linear(in_features=256, out_features=10, bias=True)
  (softmax): Softmax(dim=-1)
)`

# Result
- 1回目
バリテーション正解率,バリテーション損失,テスト正解率
`──────────────────────────
     Validate metric           DataLoader 0
──────────────────────────
         val/acc            0.9759259223937988
        val/loss            1.4837987422943115
──────────────────────────
...
──────────────────────────
       Test metric             DataLoader 0
──────────────────────────
        test/acc             0.996666669845581
──────────────────────────`

混同行列
individualImage

2~5回目は計算中

### CODE REVIEW

**Comment by miuratakumiF4** [comment]
main.pyのisortがクリアできていません。原因がわかる方は教えていただきたいです。
現状はコメントに`isort:skip_file`をつけることでisortを回避しています。

**Comment by miuratakumiF4** [comment]
> main.pyのisortがクリアできていません。原因がわかる方は教えていただきたいです。 現状はコメントに`isort:skip_file`をつけることでisortを回避しています。

解決しました。
